modules/fedimint-ln-client/src/lib.rs

```diff
@@ -908,7 +909,8 @@ impl LightningClientModule {
             .get_value(&MetaOverridesKey {})
             .await
         {
-            if cache.fetched_at.elapsed().unwrap() < META_OVERRIDE_CACHE_DURATION {
+            let elapsed = now().duration_since(cache.fetched_at).unwrap();
```

Comment:
[nit] Since this function returns a `Result`, might as well convert the `unwrap`

```suggestion
            let elapsed = now().duration_since(cache.fetched_at)?;
```